circle.yml

```diff
@@ -25,7 +25,7 @@ dependencies:
 
 test:
   override:
-    - make check
+    - make check || sudo bash -c 'source ~/extras/otp/19.0/activate && make check'  
```

Comment:
so the first check will fail with eperm, but it will fetch all the deps.  otherwise we fail to fetch deps as sudo.  i couldn't find a way to have rebar prefetch dependencies.